sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/sampleoverview/samplelist/ProjectSamplesView.groovy

```diff
@@ -39,6 +39,10 @@ class ProjectSamplesView extends VerticalLayout {
         return presenter
     }
 
+    void reset(){
+        viewModel.samples.clear()
+    }
```

Comment:
Please provide JavaDoc (as you use this method more often you could think of adding an interface `Resettable` like we did in the offer manager)